Frontend/ui-common/src/Engineering/Helpers/JBridge.js

```diff
@@ -600,27 +600,14 @@ export const updateRouteMarker = function (data) {
   }
 }
 
-export const updateRoute = function (data) {
-  return function (destMarker) {
-    return function (eta) {
-      return function (srcMarker){
-        return function (specialLocation) {
-          return function () {
-            if (window.JBridge.updateRoute) {
-              var json = JSON.stringify(data);
-              try{
-                console.log("I AM HERE ------------------ IN UPDATE ROUTE");
-                return window.JBridge.updateRoute(json, destMarker, eta, srcMarker, JSON.stringify(specialLocation));
-              }catch (err){
-                console.log("Catch error" + err);
-                return window.JBridge.updateRoute(json, destMarker, eta, JSON.stringify(specialLocation));
-              }
-            }
-          }
-        };
-      };
-    };
-  };
+export const updateRoute = (configObj) => {
+  if (window.JBridge.updateRoute) {
+    try{
+          return window.JBridge.updateRoute(JSON.stringify(configObj));
+      } catch(err){
```

Comment:
@dhruvSHA256 adding additional parameter to the preexisting function would break for the users with old app. 
Create another function so that backward compatibility is handled